core/jvm/src/test/scala/org/scalacheck/GenSpecification.scala

```diff
@@ -291,6 +291,18 @@ object GenSpecification extends Properties("Gen") with GenSpecificationVersionSp
     }
   }
 
+  property("pick produces enough distinct combinations") = {
+    val (n, nElements, nCombinations) = (3, 5, 10)
+
+    val genPick = pick(n, 1 to nElements)
+      // not interested in different permutations, only in distinct combinations
+      .map(_.toList.sorted)
```

Comment:
Out of curiosity, what is the reason for converting the output of `pick` to `List` before sorting it? It doesn't seem it is supposed to improve performance or affects the test in any other way.

Reply:
It looks like a leftover from my attempts at writing this test. Removed it.

I think, at some point I had sets and also tried to collect the stats on those. It went into `.toString` on sets which, amusingly, may be different for equal small sets (up to 4 elements). Thus, it ended up with `aSet.toList.sorted` and later got scrapped, but `toList` somehow survived 😅